### DISCUSSION TITLE
How to hide prompt in default dmenu mode

### DISCUSSION THREAD

**Discussion by fritzrehde** [davatorium/rofi#1706] [Q&A]
I would like to only see the prompt when using `rofi -dmenu -p <prompt>`. However, `rofi -dmenu` also shows the prompt "dmenu", which I don't want. I would like there to be no prompt by default, and only one if specified with `-p`. Is this possible?

**Comment by DaveDavenport** [comment]
no, you cannot hide the dmenu prompt 'by default'.. you can disable it via a flag.

**Reply by fritzrehde** [reply]
How do I disable it via a flag?

**Reply by DaveDavenport** [reply]
depends a bit on the theme if this is sufficient:
```bash
rofi -dmenu -theme-str 'prompt,textbox-prompt-colon { enabled: false; }'
```

hmm did you delete a comment?  github gave weird errors trying to reply.. please don't do that.

**Reply by fritzrehde** [reply]
Thank you for the answer. Yes, I did delete my original comment, sorry for that. I accidentally wrote it into the answer field instead of the reply one, so I thought deleting it would make it more understandable.
